src/java/org/apache/cassandra/schema/TableMetadata.java

```diff
@@ -716,6 +716,33 @@ private boolean equalsWithoutColumns(TableMetadata tm)
             && triggers.equals(tm.triggers);
     }
 
+    public boolean equalsWithoutTableNameAndDropCns(TableMetadata tm)
```

Comment:
@Maxwell-Guo This method is used only in tests. Is there some reason we can not have this function in tests only and keep the production code unchanged?